src/components/Submit/index.tsx

```diff
@@ -25,18 +25,26 @@ const Submit = ({
   tooltipBaseId,
   variant = 'success',
 }: SubmitProps): JSX.Element => {
-  const { invalid, submitting, hasValidationErrors, dirtySinceLastSubmit } =
-    useFormState({
-      subscription: {
-        dirtySinceLastSubmit: true,
-        hasValidationErrors: true,
-        invalid: true,
-        submitting: true,
-      },
-    })
+  const {
+    invalid,
+    submitting,
+    hasValidationErrors,
+    dirtySinceLastSubmit,
+    touched,
+  } = useFormState({
+    subscription: {
+      dirtySinceLastSubmit: true,
+      hasValidationErrors: true,
+      invalid: true,
+      submitting: true,
+      touched: true,
+    },
+  })
+  const isLoading = !touched || Object.keys(touched).length === 0
```

Comment:
What happens on form where we don't need to input values ?

We have some form that only require a name which can be auto generated, meaning the user only have to click submit :)

Reply:
https://github.com/scaleway/scaleway-form/pull/378#issuecomment-1175106427

Reply:
I don't think so but might need to check that :)